kit/wifi.md

```diff
@@ -6,69 +6,56 @@ title: WiFi
 
 You can now connect to your robot in the same way you normally connect to a WiFi network.
-You will need a WiFi key to be able to connect and you can find this inside any robot.zip
-that you have exported from the IDE. Just unzip the robot.zip and have a look inside `wifi.yaml`.
+You will need a WiFi key to be able to connect and you can find this inside `robot-settings.toml` on the USB drive containing your code.
 
-Note that, because the information for your robot's WiFi network is stored inside the robot.zip,
-the WiFi network will disappear when you unplug the USB memory stick. It will reappear a few moments
-after you plug the USB memory stick into your robot.
+These details can also be printed using:
+~~~~ python
+R.print_wifi_details()
+~~~~
 
-If you are having any problems connecting to your robot, just head on over to the [forum]({{ site.url }}/forum)
+If you are having any problems connecting to your robot, just head on over to the Discord
 and ask for help.
 
-Interacting With Your Robot
----------------------------
+Using the robot interface
+-------------------------
 
-Once you have a WiFi connection, visit `http://robot.sr` in a web browser to see the robot interface.
+Once you have a WiFi connection, visit `http://robot.lan` in a web browser to see the robot interface.
 
 The robot interface gives you the ability to remotely start the code on your robot,
 as well as view the logs.
 
-<div class="info" markdown="1">
-The stop button that appears after starting your code is currently not functional.
-</div>
+![The robot interface on desktop]({{ site.baseurl }}/images/content/kit/wifi_interface.png)
 
-### Setting up the Robot's Environment
+### Viewing logs
 
-![The robot interface's set up screen]({{ site.baseurl }}/images/content/kit/remote-interface/set-up-screen.png)
+The robot interface allows you to see all messages from your code sent by `print(...)`
+statements (and anything else that outputs to standard output or standard
+error). It will also show messages from the initialisation of the robot's
+hardware, as well as any errors that occurred when running your code.
 
-When you first navigate to the robot interface you are presented with the Set up page.
-This page allows you to select the starting zone of your robot, as well as
-whether it is running in development mode or competition mode. Changing the
-starting zone allows you to test how your robot handles being started in a
+You can also see your robot's logs on the USB stick, in a file called
+`log.txt`.
+
+### Setting up the Robot's Environment
+
+The toolbar present at the top of the page, or bottom of the page on mobile, allows you to change the robot's starting zone and mode.
+Changing the starting zone allows you to test how your robot handles being started in a
 different zone.
 
+The wifi interface will only be fully accessible when your robot is started in dev mode and will be restricted in competition mode.
```

Comment:
Nit:
```suggestion
The WiFi interface will only be fully accessible when your robot is started in dev mode and will be restricted in competition mode.
```

Reply:
Is the interface actually restricted?

Reply:
Just having `COMP` mode on doesn't disable it. The competition mode USB sets `wifi_enabled` in the Astoria metadata to be `false`.

Reply:
Ahh ok! A tad nuclear as won't be able to see logs but it does solve the issue of restricting controls. Maybe we can look at allowing some access to wifi once we have some headroom

Reply:
That was the idea of "arena connectivity".